src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigCommand.cs

```diff
@@ -9,6 +9,7 @@ namespace NuGet.CommandLine.XPlat
 {
     internal class ConfigCommand
     {
+        public const string WORKING_DIRECTORY = "WORKING_DIRECTORY";
```

Comment:
I believe only this class references this constant, in which case this should be private.

```suggestion
        private const string WORKING_DIRECTORY = "WORKING_DIRECTORY";
```